scripts/scraper-ng/rules/html-require-recipe-ingredients/ingredient-handlers/data-constructor.js

```diff
@@ -26,11 +41,20 @@ function handleDataConstructor(tree, logger) {
     return null;
   }
 
+  const section = utils.sliceSection(heading, tree);
+
+  // Constructor sections are allowed to have no actual links
+  // to constructors, if they explicitly record this fact
+  let ok = checkNoConstructor(section.children);
```

Comment:
A small style question here: Is the checkNoConstructor call significant after this line? I'm not sure why it shares a variable with the result of `checkLinkList`. Is there a reason you didn't go with `if (checkNoConstructor(…))` on the next line?

Reply:
No, you're right, that is simpler and I've made the change.